packages/remix-server-adapter/src/defaultRemixConfig.ts

```diff
@@ -0,0 +1,8 @@
+import type { AppConfig } from '@remix-run/dev'
```

Comment:
I went ahead with your proposal for default app config from adapters @ascorbic https://github.com/remix-run/remix/discussions/4878